.pre-commit-config.yaml

```diff
@@ -420,12 +420,14 @@ repos:
     rev: '1e78a9acf3110e1f9721feb591f89a451fc9876a'
     hooks:
       - id: isort
+        exclude: ^salt/ext/.*$
```

Comment:
already found here: https://github.com/saltstack/salt/blob/master/.isort.cfg

But I would totally approve a change moving from .isort.cfg to [pyproject.toml](https://github.com/timothycrosley/isort/issues/694#issuecomment-581246882)

Reply:
Doesn't hurt here too, pre commit just doesn't even send these to black or isort.

Reply:
Would you agree?

Reply:
I think if we leave them here it will be confusing if we need to make changes.

There should be one-- and preferably only one --obvious place to configure the tools that we're using.

I think putting comments in here, like `# See pyproject.toml for excludes` will be enough. And not misleading, since I would expect this exclude to do something.